content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -146,20 +146,6 @@ All code snippets in Windows sections are to be run in a PowerShell environment
 with elevated permissions (Administrator) on the Windows worker node.
 {{< /note >}}
 
-{{< tabs name="tab-windows-kubeadm-runtime-installation" >}}
-{{% tab name="Docker EE" %}}
-
-#### Install Docker EE
-
-Install the `Containers` feature
-
-```powershell
-Install-WindowsFeature -Name containers
-```
-
-Install Docker
-Instructions to do so are available at [Install Docker Engine - Enterprise on Windows Servers](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server#install-docker).
-
```

Comment:
This tab actually starts on line 150 and continues to line 162.

Instead of removing one of the 2 tabs, I would tell readers that they must install a container runtime and list some compatible ones. Rather than providing the instructions, we hyperlink to them.

For example:
- containerd
- Mirantis Container Runtime

Reply:
We can also link to https://github.com/kubernetes-sigs/sig-windows-tools/ to let readers know that there might be a helper script for your preferred runtime.

Reply:
+1 to this suggestion